Package.swift

```diff
@@ -22,8 +22,8 @@ let package = Package(
   dependencies: [
     .package(
       name: "abseil",
-      url: "https://github.com/firebase/abseil-cpp-SwiftPM.git",
-      .revision("05d8107f2971a37e6c77245b7c4c6b0a7e97bc99")
+      url: "https://github.com/yulin-liang/abseil-cpp.git",
+      .revision("a8fca021df775593b6bb2e7c865be6a930877b66")
```

Comment:
That's because we also need to support swift package manager for the latest abseil, Firebase's abseil is not the latest version, we can not depend it. I'll change it back to Firebase abseil and add a TODO here.

Reply:
Done.